spec/fixtures/site/_config.yml

```diff
@@ -0,0 +1,4 @@
+collections:
+  things:
+    output: true
+    permalink: /:path/
```

Comment:
This plugin is currently opti-in, correct?

Also, one way to speed this up would be to generate a cache of `path` => `url` for all pages/collections once and then use a hash to map it instead of looping over all records for every link. It'll be `O(n)` instead of `O(n^2)`.